lua/autorun/client/cl_focus.lua

```diff
@@ -0,0 +1,26 @@
+local tabbedOutList = {}
+local isTabbedOut = false
+
+timer.Create( "CFC_AttentionMonitor_tabNetTimmer", 1, 0, function()

```

Comment:
Considering you changed the logic and made it only send when there is an actual change you could change the timer interval to lower like 0.1 if needed